src/common.rs

```diff
@@ -161,8 +161,7 @@ impl Errors {
 #[derive(Derivative)]
 #[derivative(Default(bound = ""))]
 pub struct Merge<F> {
-    #[derivative(Default(value = "PhantomData"))]
-    future_type: PhantomData<F>,
+    #[derivative(Default(value = "PhantomData"))] future_type: PhantomData<F>,
```

Comment:
Oh, is this some new rustfmt setting again?

Reply:
Is this attribute necessary at all?

Reply:
Yes, otherwise `F` would have to implement `Default`.

Reply:
No. PhantomData implements default for any type parameter.

Reply:
It does now but I think it was a rather recent fix which I don't want to rely on yet.

Reply:
It does since 1.0.0

Reply:
Ahhh, the attribute on `PhantomData` isn't required, but we need `derivative` here.